src/main.cpp

```diff
@@ -4,19 +4,22 @@
 #include <string>
 
 #include "mirco_evaluate.h"
+#include "mirco_filesystem_utils.h"
+#include "mirco_postprocess.h"
 #include "mirco_setparameters.h"
 #include "mirco_topology.h"
 #include "mirco_topologyutilities.h"
 
-
 int main(int argc, char* argv[])
 {
-  TEUCHOS_TEST_FOR_EXCEPTION(
-      argc != 2, std::invalid_argument, "The code expects (only) an input file as argument");
+  TEUCHOS_TEST_FOR_EXCEPTION(argc != 3, std::invalid_argument,
+      "The code expects input file with a full path and a prefix for output files.");
   // reading the input file name from the command line
   std::string inputFileName = argv[1];
+  std::string outputFileName = argv[2];
 
-  const auto start = std::chrono::high_resolution_clock::now();
+  // following function generates the actual path of the output file.

```

Comment:
```suggestion
  // Generate the <...> path to the output file
```
... replace `<...>` with absolute or relative.